### PR TITLE
Move PEC to Enterprise Edition

### PR DESCRIPTION
**Pull request by nawroth**
Note comment by @thobe:
We also still need to address these two issues:
https://github.com/neo4j/neo4j/pull/5200#discussion_r37968974
https://github.com/neo4j/neo4j/pull/5200#discussion_r37969000

### CODE REVIEW

**Comment by thobe** [comment]
The `For community server, use the community launcher` commit addresses the two comments I made on #5200.

File: enterprise/kernel/pom.xml [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+    </plugins>
+  </build>
+
+  <dependencies>
+    <!-- Neo4j dependencies -->
+    <dependency>
+      <groupId>org.neo4j</groupId>
+      <artifactId>neo4j-advanced</artifactId>
+      <version>${project.version}</version>
+    </dependency>
```
Comment (davidegrohmann) [inline enterprise/kernel/pom.xml:90]: Why?

Comment (thobe) [inline enterprise/kernel/pom.xml:90]: It references all the components that make sense here. What would you like the alternative to be? just `neo4j-kernel`?

Comment (davidegrohmann) [inline enterprise/kernel/pom.xml:90]: Isn't better to list "all the components that make sense here" explicitly instead? Yes I would expect only `neo4j-kernel` here.

File: enterprise/kernel/pom.xml [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+    <dependency>
+      <groupId>org.neo4j</groupId>
+      <artifactId>neo4j-cypher</artifactId>
+      <version>${project.version}</version>
+    </dependency>
```
Comment (davidegrohmann) [inline enterprise/kernel/pom.xml:95]: This looks like it should be test only.

**Comment by systay** [comment]
Addressing the comments in a separate PR, to make sure this is in

**Pull request merged**